platform/core/PolyPodCoreAndroid/gradle.properties

```diff
@@ -0,0 +1,23 @@
+# Project-wide Gradle settings.
+# IDE (e.g. Android Studio) users:
+# Gradle settings configured through the IDE *will override*
+# any settings specified in this file.
+# For more details on how to configure your build environment visit
+# http://www.gradle.org/docs/current/userguide/build_environment.html
+# Specifies the JVM arguments used for the daemon process.
+# The setting is particularly useful for tweaking memory settings.
+org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
```

Comment:
Couldn't you add some "UnsignedExperimentalWhatever" here to not need to insert it in specific files?
Also, what's the best practice regarding these experimental features?

Reply:
These are the JVM parameters, i.e. runtime. The `ExperimentalUnsignedTypes` parameter is an option for the Kotlin compiler. So no :D